### DISCUSSION TITLE
Unable to get Python support

### DISCUSSION THREAD

**Discussion by Raj2032** [VSCodium/vscodium#1446] [Q&A] [answered]
I am using Arch-based Linux distro and installed this specific extension:

![image](https://user-images.githubusercontent.com/29293830/226093214-9900bc3a-c728-40e2-8e9d-68ab03be0290.png)

However it doesn't show any errors that I have made at all. I am using `Python 3.10.10` on my system and it came with the OS. Any idea as to why this has happened?

**Comment by Mr-Spade** [comment, marked as answer]
This is because the extension "Pylance" which provide syntax highlight and error check is not open source.

For an alternative, you can use "pyright" for error check, and "Syntax Highlighter" for syntax highlight.
